webdriver/tests/element_click/link.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+
+from tests.support.asserts import assert_error, assert_success
+from tests.support.inline import inline
+
+def click(session, element):
+    return session.transport.send("POST", "session/{session_id}/element/{element_id}/click"
+                                  .format(session_id=session.session_id,
+                                          element_id=element.id))
+
+
+# 14.1 Element Click Link Element Tests Step 7
+
+@pytest.mark.parametrize("element", ["<a href=\"/webdriver/tests/element_click/support/input.html\">123456</a>",
+                                     """<p style="background-color: rgb(255, 255, 0); width: 5em;">
+                                        <a href="/webdriver/tests/element_click/support/input.html">Helloooooooooooooooooooo Worlddddddddddddddd</a>
+                                    </p>"""])
```

Comment:
Please don’t use parameterisation for this. It makes the test almost impossible to understand.

Reply:
Do you want a test for each of the inputs? The test is the same with just 3 different input suggested by David.

Reply:
fixed

Reply:
I can’t see that this is fixed?

Reply:
Sorry, I thought I push this change. Fixed now.